acceptance/static_roles_test.go

```diff
@@ -42,17 +42,36 @@ func extractStaticRoleData(t *testing.T, resp *http.Response) *staticRoleData {
 func (p *PluginTest) TestStaticRoleLifecycle() {
 	t := p.T()
 
-	cloud := &openstack.OsCloud{
-		Name:             openstack.RandomString(openstack.NameDefaultSet, 10),
-		AuthURL:          "https://example.com/v3",
-		UserDomainName:   openstack.RandomString(openstack.NameDefaultSet, 10),
-		Username:         openstack.RandomString(openstack.NameDefaultSet, 10),
-		Password:         openstack.RandomString(openstack.PwdDefaultSet, 10),
-		UsernameTemplate: "u-{{ .RoleName }}-{{ random 4 }}",
+	cloud := openstackCloudConfig(t)
+	require.NotEmpty(t, cloud)
+
+	client, aux := openstackClient(t)
+
+	dataCloud := map[string]interface{}{
+		"auth_url":         cloud.AuthURL,
+		"username":         cloud.Username,
+		"password":         cloud.Password,
+		"user_domain_name": cloud.UserDomainName,
 	}
-	p.makeCloud(cloud)
 
-	data := expectedStaticRoleData(cloud.Name)
+	resp, err := p.vaultDo(
+		http.MethodPost,
+		cloudURL(cloudName),
+		dataCloud,
+	)
+	require.NoError(t, err)
+	assert.Equal(t, http.StatusNoContent, resp.StatusCode, readJSONResponse(t, resp))
+
+	createUserOpts := users.CreateOpts{
+		Name:        "vault-test",
+		Description: "Static user",
+		DomainID:    aux.DomainID,
+		Password:    openstack.RandomString(openstack.PwdDefaultSet, 16),
+	}
+	user, err := users.Create(client, createUserOpts).Extract()
+	require.NoError(t, err)
```

Comment:
Isn't there a test-specific function for user creation?

Reply:
Yeah but it's using dynamic role and its user_roles/user_groups inside logic so I don't think it will suite us

Reply:
Does it require big changes to be reused here?

Reply:
Yeah I think so